client/pages/sources/show/format/format.html

```diff
@@ -60,5 +61,53 @@
         {{/if}}
       </aside>
     </section>
+    {{else}}
+    <section class='page-content centered load-default-json'>
+      <div class='block'>
+        <h1>Select your data format</h1>
+        <p>
+          <form>
+          <table class='formats'>
+            <tbody>
+              <tr>
+                <td><label><input name='format' type='radio' value='csvWithoutHeader'>&nbsp;CSV without header</label></td>
```

Comment:
NABD but it might make sense to use an `{{each}}` here and iterate over an array to build these `<tr>`s